lab-chris/lib/server.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const http = require('http');
+const router = require('./router.js');
+const Player = require('../model/player.js');
+
+var storage = {};
+
+router.post('/api/player', (req, res) => {
+  if(!req.body.name){
```

Comment:
What would happen if I sent a `name` in the body but not a `team` or `position`?